pkg/kubelet/container/runtime.go

```diff
@@ -46,12 +46,10 @@ type Runtime interface {
 	GetPods(all bool) ([]*Pod, error)
 	// RunPod starts all the containers of a pod within a namespace.
 	RunPod(*api.Pod, map[string]volume.Volume) error
+	// Syncs the running pod into the desired pod.
+	SyncPod(pod *api.Pod, runningPod Pod, podStatus api.PodStatus) error
```

Comment:
I think we should pass runningPod to SyncPod. The running pod is from pod workers. We might need to refactor that, but for now it's efficient to pass that to SyncPod I think?
@vmarmol @yujuhong